src/lib.rs

```diff
@@ -363,18 +362,15 @@ impl KVMIntrospectable for KVMi {
         Ok(())
     }
 
-    fn get_page_access(&self, gpa: u64) -> Result<KVMiPageAccess, Error> {
```

Comment:
Why is `get_page_access` removed without a replacement?

Reply:
Because it has been removed in libkvmi v7:
https://github.com/bitdefender/libkvmi/blob/master/include/libkvmi.h

I can't remember the design decision behind this, but you can ask on kvm-vmi slack

Reply:
But `get_page_access` is defined by our API. This means that we will now panic with `unimplemented` if the user tries to call this function. I am sure that there is an approach that will yield the same results which we should try to implement instead of dropping the function completely.

Reply:
We should maintain a hashtable of modified page permissions and return them, as what libvmi does:
https://github.com/libvmi/libvmi/blob/master/libvmi/events.c#L667

There is not `get_mem_access_ptr` defined in libvmi's driver interface:
https://github.com/libvmi/libvmi/blob/master/libvmi/driver/driver_interface.h